crates/ide-diagnostics/src/tests.rs

```diff
@@ -334,3 +334,43 @@ fn minicore_smoke_test() {
     eprintln!("Checking all minicore flags");
     check(MiniCore::from_flags(MiniCore::available_flags(MiniCore::RAW_SOURCE)))
 }
+
+#[test]
+fn crate_attrs_smoke_test() {
```

Comment:
We really prefer to not add new tests here. Perhaps there should be a place for testing of lint attributes, but for now stick them under any handler they are tested in.